include/Metabox/Posts/Category.php

```diff
@@ -31,26 +31,38 @@ public function render() {
 				<tr>
 					<td scope="row" >
 						<?php
-						wp_dropdown_categories( array(
-							'name'         => 'smbm_mc_selected_cat',
-							'show_count'   => true,
-							'hierarchical' => true,
-							'orderby'      => 'NAME',
-							'hide_empty'   => false,
-						) );
+						$categories = get_categories( array(
```

Comment:
Don't duplicate code. Create this as a helper function in the `BM_Metabox_PostBase` class